vector/src/main/java/im/vector/app/core/pushers/VectorPushHandler.kt

```diff
@@ -142,11 +143,6 @@ class VectorPushHandler @Inject constructor(
         pushData.roomId ?: return
         pushData.eventId ?: return
 
-        // If the room is currently displayed, we will not show a notification, so no need to get the Event faster
-        if (notificationDrawerManager.shouldIgnoreMessageEventInRoom(pushData.roomId)) {
```

Comment:
I guess we do not have the threadId in the `pushData`, true?

Reply:
We don't :(